### ISSUE TITLE
Editor hover should support PgUp, PgDown, Home and End navigation

### ISSUE THREAD

**Issue by LiamKearn** [microsoft/vscode#139717]
This would be nice for Accessibility for those of us using keyboards.

Currently when I use `editor.action.showHover` my focus is moved the the hover and I can tab around to any anchor tags etc. that are present but when I use: PgUp, PgDown, Home, End and the arrow keys I cannot scroll the infomation provided in the hover.

I've tried with and without the setting:
`editor.accessibilitySupport`

I have no clue how javascript works but I suspect it might have something todo with the fact that the content is split into two:
`.hover-row .markdown-hover` elements.

I will continue to fill out this feature request whilst I take a look.

Thanks

**Comment by isidorn** [comment]
@LiamKearn sorry for the slow response. This sounds like a feature request that we support PgUp, PgDown, Home and End navigation in the Editor Hover. I think it is a fair request.

**Comment by LiamKearn** [comment]
@isidorn Yeah, a feature request sounds more appropriate. Thanks for the reply.

Happy to close at your discretion or rewrite with a feature template if you think it might have a chance of being picked up.

**Comment by isidorn** [comment]
@LiamKearn no need for that. Let's give the Henning and Alex time to consider this feature request and we'll see how it goes.

**Comment by LiamKearn** [comment]
Alrighty thanks again!

**Comment by abrahamadamu** [comment]
Yeah, really important feature and very easy to implement. Also stated [here](https://github.com/microsoft/vscode/issues/167264)

**Comment by isidorn** [comment]
fyi @meganrogge it would be cool if we supported keyboard to scroll both for hover and parameter hints (or for all editor content widgets in general).

**Comment by aiday-mar** [comment]
Hi thanks for filing this issue. This issue should be resolved as of today. Let me know if you are not able to use the keys to navigate the hover.

**Comment by musjj** [comment]
I can't figure out a way to scroll the parameter hints using keyboard shortcuts. Is there a specific command/method for this?

**Comment by aiday-mar** [comment]
Hi @musjj the parameter hints is not a standard editor hover, it does not seem to have keybindings attached to the scrolling. Pinging @mjbvz about this.
